docs/user-guide/cdf_attribute_management.rst

```diff
@@ -141,16 +141,153 @@ Each of the keys for variable metadata requirements are defined in the table bel
+
+.. code-block:: yaml
+
+    instrument_base: &instrument_base
+      Descriptor: GLOWS>GLObal Solar Wind Structure
+      TEXT: >
+        The GLObal Solar Wind Structure (GLOWS) is a non-imaging single-pixel Lyman-alpha
+        photometer to investigate the global heliolatitudinal structure of the solar wind
+        and its evolution during the solar cycle. Additionally, GLOWS investigates the
+        distribution of interstellar neutral hydrogen (ISN H) and the solar radiation
+        pressure acting on ISN H. The objectives of GLOWS are accomplished by observation
+        of the modulation of heliospheric backscatter glow of ISN H (the helioglow)
+        along a scanning circle in the sky.
+        GLOWS design and assembly is led by the Space Research Center, Warsaw, Poland
+        (CBK PAN). See https://imap.princeton.edu/instruments/glows for more details.
+      Instrument_type: Imagers (space)
+
+    imap_glows_l1a_hist:
+      <<: *instrument_base
+      Data_level: L1A
+      Data_type: L1A_hist>Level-1A histogram
+      Logical_source: imap_glows_l1a_hist
+      Logical_source_description: IMAP Mission GLOWS Histogram Level-1A Data.
+
+    imap_glows_l1a_de:
+      <<: *instrument_base
+      Data_level: L1A
+      Data_type: L1A_de>Level-1A direct event
+      Logical_source: imap_glows_l1a_de
+      Logical_source_description: IMAP Mission GLOWS Direct Event Level-1A Data.
+
+
+These global attributes can be added to an instance of cdf_attribute_manager and then retrieved and validated:
+
+.. code-block:: python
+
+    shared_global_attributes = Path("shared_global_attributes.yaml")
+    instrument_global_attributes = Path("instrument_global_attributes.yaml")
+
+    cdf_manager = CdfAttributeManager(use_defaults=True)
+
+    # Load in the global attributes
+    cdf_manager.load_global_attributes(shared_global_attributes)
+    cdf_manager.load_global_attributes(instrument_global_attributes)
+
+    # retrieve the global attributes, including the specific GLOWS L1A Histogram attributes
+    global_attrs = cdf_manager.get_global_attributes(instrument_id="imap_glows_l1a_hist")
+
+
+Variable attribute files work similarly to the instrument ID. Each variable has a name assigned to it, which then has a set of attributes associated with it. YAML anchors and aliases are used to create
+defaults and shared information. Then, the variable attributes are retrieved with the name.
+
+
+.. code-block:: yaml
+
+    int_fillval: &int_fillval -9223372036854775808
+
+    default_attrs: &default_attrs
+      DISPLAY_TYPE: no_plot
+      TIME_BASE: J2000
+      TIME_SCALE: Terrestrial Time
+      REFERENCE_POSITION: Rotating Earth Geoid
+      FILLVAL: *int_fillval
+
+    support_data_defaults: &support_data_defaults
+      <<: *default_attrs
+      DEPEND_0: epoch
+      VALIDMIN: 0
+      VALIDMAX: 1
+      DISPLAY_TYPE: time_series
+      VAR_TYPE: support_data
+      FORMAT: I10
+      RESOLUTION: ISO8601
+
+    bins_attrs:
```

Comment:
Maybe this was mentioned and I missed it, but maybe add an example showing you can stack defaults?
```yaml
bins_attrs:
       <<: *default_attrs
       <<: *bin_defaults
       VALIDMAX: 3599
       CATDESC: Histogram bin number
       FIELDNAM: Bin number
       FORMAT: I5
       LABLAXIS: Counts
       FILLVAL: -32768
       MONOTON: INCREASE
       SCALETYP: linear
```